dbt_platform_helper/utils/application.py

```diff
@@ -125,7 +125,7 @@ def is_environment_key(name):
 
 def get_application_name(abort=abort_with_error):
     if Path(PLATFORM_CONFIG_FILE).exists():
-        config = ConfigProvider()
+        config = ConfigProvider(installed_platform_helper_version="14.0.0")
```

Comment:
Do we want this version hard coded? I think updating it might get missed.

Reply:
I don't remember why I even put this in. 🤔

Reply:
Urgh - it was to get around the fact that if you are pointing at a local version of platform-helper or running the tests, the importlib.version method can't find the installed version of platform-helper and so errors. I'll think of a better way than this...